wire-ios-data-model/Source/Repositories/FeatureRepository.swift

```diff
@@ -334,7 +334,8 @@ public class FeatureRepository: FeatureRepositoryInterface {
 
         switch e2ei.status {
         case .enabled:
-            notifyChange(.e2eIEnabled)
+            let gracePeriod = TimeInterval(e2ei.config.verificationExpiration)
```

Comment:
**suggestion:** No need to store this value in a separate variable. We could directly pass it.